schema/03-staking.sql

```diff
@@ -82,3 +77,15 @@ CREATE TABLE validator_voting_power
     height            BIGINT NOT NULL,
     PRIMARY KEY (consensus_address, height)
 );
+
+CREATE TABLE validator_description
+(
+    operator_address      TEXT NOT NULL,
```

Comment:
@HarleyAppleChoi Can we add `references validator_info(operator_address)` here? This would allow Hasura to automatically create a relationship and include the `description` field inside the `validator_info` object. 

```suggestion
    operator_address      TEXT NOT NULL REFERENCES validator_info(operator_address),
```